lab-mark/__test__/VideoGame-router.test.js

```diff
@@ -0,0 +1,181 @@
+'use strict';
+
+// mock env
+process.env.PORT = 7000;
+process.env.MONGODB_URI = 'mongodb://localhost/testing';
+
+const faker = require('faker');
+const superagent = require('superagent');
+const VideoGame = require('../model/VideoGame.js');
+const server = require('../lib/server.js');
+
+const apiURL = `http://localhost:${process.env.PORT}`;
+
+const videogameMockCreate = () => {
+  return new VideoGame({
+    title: faker.lorem.words(10),
+    genre: faker.lorem.words(10),
+    console: faker.lorem.words(10),
+  }).save();
+};
+
+let videogameMockMany = (num) => {
+  return Promise.all(new Array(num).fill(0).map(() => videogameMockCreate()));
+};
+
+describe('/api/videogames', () => {
+  beforeAll(server.start);
+  afterAll(server.stop);
+  afterEach(() => VideoGame.remove({}));
+
+  describe('POST /api/videogames', () => {
+    test('should respond with a videogame and 200 status', () => {
+      let tempVideoGame = {
+        title: faker.lorem.words(10),
+        genre: faker.lorem.words(10),
+        console: faker.lorem.words(10),
+      };
+      return superagent.post(`${apiURL}/api/videogames`)
+        .send(tempVideoGame)
+        .then(res => {
+          expect(res.status).toEqual(200);
+          expect(res.body._id).toBeTruthy();
+          expect(res.body.timestamp).toBeTruthy();
+          expect(res.body.title).toEqual(tempVideoGame.title);
+          expect(res.body.console).toEqual(tempVideoGame.console);
+          expect(res.body.genre).toEqual(tempVideoGame.genre);
+        });
+    });
+
+    test('should respond with a 400 status', () => {
```

Comment:
Try adding "when/if ..." at the end of your descriptions. It makes tests easier to understand for a third-party. 

Example: "Should respond with a 400 status when the body is invalid"